suse2013/xhtml/verbatim.xsl

```diff
@@ -14,7 +15,26 @@
     exclude-result-prefixes="exsl">
 
 <xsl:template match="programlisting|screen|synopsis|computeroutput|userinput|literallayout">
-<div class="verbatim-wrap">
+<!-- This list is intentionally quite strict. "Be strict in what you accept, be
+     li{b,t}eral in what you output." (Or something.) In any case, this should
+     hopefully keep the documents consistent -->
+<xsl:variable name="supported" select="'|apache|bash|c++|css|diff|html|xml|http|ini|json|java|javascript|makefile|nginx|php|perl|python|ruby|sql|crmsh|dockerfile|lisp|yaml|'"/>
```

Comment:
Shouldn't we move that into `param.xsl` and give them a descriptive name?

Although we probably don't change that very often, I think it would better declare it as `xsl:param` to allow user customization (especially if our stylesheets are used as a basis for customization).